src/main/java/org/esupportail/portlet/filemanager/portlet/CustomDispatcherPortlet.java

```diff
@@ -0,0 +1,32 @@
+package org.esupportail.portlet.filemanager.portlet;
+
+import javax.portlet.MimeResponse;
+import javax.portlet.PortletRequest;
+import javax.portlet.PortletRequestDispatcher;
+
+import org.springframework.web.portlet.DispatcherPortlet;
+
+public class CustomDispatcherPortlet extends DispatcherPortlet {
+
+    /**
+     * Perform a dispatch on the given PortletRequestDispatcher.
+     * <p>The default implementation uses a forward for resource requests
+     * and an include for render requests.
+     * @param dispatcher the PortletRequestDispatcher to use
+     * @param request current portlet render/resource request
+     * @param response current portlet render/resource response
+     * @throws Exception if there's a problem performing the dispatch
+     */
+    @Override
+    protected void doDispatch(PortletRequestDispatcher dispatcher, PortletRequest request, MimeResponse response)
+            throws Exception {
+        // Comment fix code as getting this problem https://github.com/spring-projects/spring-framework/issues/15417
+        // TODO need to rebuild the portlet => WebComponent + API to fix
+//        if (PortletRequest.RESOURCE_PHASE.equals(request.getAttribute(PortletRequest.LIFECYCLE_PHASE))) {
```

Comment:
Is this commented code going to be needed, could it be removed?

Reply:
The commented code is the original one to remember what was there before. 
In my mind rebuilding the API and the frontend is needed and so this code will be removed. So it's in waiting better and to "document" what is done here.